src/framework/execution/error.ts

```diff
@@ -16,3 +16,13 @@ export class IntegrationConfigValidationError extends Error
   implements IntegrationError {
   code = 'LocalConfigFieldTypeMismatch';
 }
+
+export class IntegrationStepStartStateInvalidStepIdError extends Error
+  implements IntegrationError {
+  code = 'InvalidStepIdSpecifiedInStartState';
```

Comment:
```suggestion
  code = 'UnknownStepIdSpecifiedInStartState';
```
I prefer the term "Unknown" over "Invalid". What do you think, would it be more clear this way? Maybe "Undefined"?

Reply:
Going with "Unknown"